src/BuildScriptGeneratorCli/Commands/DockerfileCommand.cs

```diff
@@ -4,6 +4,7 @@
 // --------------------------------------------------------------------------------------------
```

Comment:
We should also include a `LogTimedEvent` call wrapping the execution of the `GenerateDockerfile` command in `DefaultDockerfileGenerator` so we can log information about the build and runtime images and tags that we detect and what their result was.